Documentation/Blazorise.Docs/Pages/Docs/Extensions/Autocomplete/AutocompletePage.razor

```diff
@@ -325,4 +325,7 @@
     <DocsAttributesItem Name="Virtualize" Type="bool" Default="false">
         Gets or sets whether the Autocomplete will use the <Code>Virtualize</Code> functionality.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="BadgeTemplate" Type="RenderFragment<BadgeContext<TItem, TValue>>">
```

Comment:
I know we use the Badge for the autocomplete tags. But I don't think we should expose that naming outside of it. The parts that are selected are usually named Tags, so maybe we should follow that convention.

Reply:
So `TagTemplate`? It still requires the user to have knowledge of what a Tag would be in this context, since we don't really mention it anywhere...
Maybe `MultipleSelectionTemplate` that would be agnostic?

Reply:
Yeah, I think `Tag` is the most practiced name in this context, Although something with `Selected` is also a good idea.

Reply:
`MultipleSelectedTemplate` ? I don't really think we can run much from that since everything is named around "`Multiple`"
![image](https://github.com/Megabit/Blazorise/assets/22283161/94cba5ce-27d2-45a9-8e64-380793b06e8f)

Even though `Tag` can be common, it would still be a guessing name, some people might know some might not. Specifially since we don't have any other mentions anywhere.

Reply:
@stsrki friendly reminder